robottelo/cli/factory.py

```diff
@@ -391,6 +392,46 @@ def make_gpg_key(options=None):
     return create_object(GPGKey, args, options)
 
 
+@cacheable
+def make_content_credential(options=None):
+    """Creates a content credential.
+
+    In Satellite 6.8 only gpg_key is supported.
+
+    :param options: Check options using `hammer content-credentials create --help` on satellite.
+
+    :returns ContentCredential object
+    """
+    # Organization ID is a required field.
+    if not options or not options.get('organization-id'):
+        raise CLIFactoryError('Please provide a valid ORG ID.')
+
+    # Create a fake gpg key file if none was provided
+    if not options.get('key'):
+        (_, key_filename) = mkstemp(text=True)
+        os.chmod(key_filename, 0o700)
+        with open(key_filename, 'w') as gpg_key_file:
+            gpg_key_file.write(gen_alphanumeric(gen_integer(20, 50)))
+    else:
+        # If the key is provided get its local path and remove it from options
+        # to not override the remote path
+        key_filename = options.pop('key')
+
+    args = {
+        'key': '/tmp/{0}'.format(gen_alphanumeric()),
```

Comment:
Consider using f-strings to generate this string.

Reply:
OK, change made